components/LoginForm.vue

```diff
@@ -56,21 +53,13 @@ import { mapActions } from 'vuex';
 export default Vue.extend({
```

Comment:
Not needed to merge this, but we should add a router guard for this at some point, to prevent access to the login form if we're not connected to a server.

Something like:
```typescript
    beforeResolve (to, from, next) {
        if (this.$store.server.isConnected) {
            next('/selectserver')
        } else {
            next();
        }
    }
```

(Very untested and I'm not sure it's right)

Reply:
I've added middleware to the select server and login pages
if there is not a base url -> select server page
if there is a user id in the store -> home